core/src/avm1/value.rs

```diff
@@ -68,6 +66,12 @@ impl<'gc> From<u32> for Value<'gc> {
     }
 }
 
+impl<'gc> From<usize> for Value<'gc> {
+    fn from(value: usize) -> Self {
+        Value::Number(value as f64)
+    }
+}
```

Comment:
This makes me a little squeamish because a 64-bit `usize` is not guaranteed to fit losslessly into an `f64`, and Rust doesn't define `From<usize> for f64`. I guess it's okay for ergonomics-sake, and `args.len()` etc. will never be greater than 32-bit in SWF land. Another option would be to use `TryFrom` and return an error if it's greater than Number.MAX_SAFE_INTEGER.